.github/ISSUE_TEMPLATE/ux_report.yaml

```diff
@@ -6,15 +6,34 @@ assignees: ["donouwens"]
 
 body:
   - type: checkboxes
-    id: urgency
+    id: impact
     attributes:
-      label: URGENCY
-      description: Is this issue blocking your progress?
+      label: IMPACT
+      description: Define or estimate how many users will be faced with this issue.
       options:
         - label: High
         - label: Medium
         - label: Low
-        - label: I don't know
+  - type: checkboxes
+    id: urgency
+    attributes:
+      label: URGENCY
+      description: What is the expected speed this needs to be resolved?
+      options:
+        - label: Critical
+        - label: Major
+        - label: Medium
+        - label: Minor
+  - type: checkboxes
+    id: priority
+    attributes:
+      label: PRIORITY
```

Comment:
Currently, the description renders a bit off?
![Screenshot from 2023-04-17 19-05-58](https://user-images.githubusercontent.com/2518544/232560282-8e3b7081-ce16-46f6-8c57-dc3905d69eb8.png)


I would perhaps just omit the priority and would prefer you assign the appropriate priority label to an issue based on IMPACT and URGENCY after it is filled?

Reply:
@donouwens, ping